webapp/src/lib/utils.js

```diff
@@ -71,3 +72,37 @@ export function distanceInWordsToNow(date) {
     locale: getCurrentLocale()
   })
 }
+
+/**
+ * Returns if a date is expired
+ * @param {number} - expires_at
+ * @returns {boolean}
+ */
+export function isExpired(expires_at) {
+  return parseInt(expires_at, 10) < Date.now()
+}
+
+/**
+ * Returns if object is Open
+ * @param {object} - obj with status & tx_status fields
+ * @param  {string} - status
+ * @returns {boolean}
+ */
+export function isOpen(obj, status) {
```

Comment:
I don't think this is necessary? I really like `hasStatus`, but the rest of the codebase ended up using `isOpen(publication, STATUS.open)` which means that I can do `isOpen(publication, STATUS.cancelled)` and it won't really be true.

So, to me, just having hasStatus is enough.

Reply:
+1 ☝️